### PR TITLE
updating readme.md

### PR DESCRIPTION
**Pull request by evandrosystems**

### CODE REVIEW

File: .idea/.gitignore
Comment (ravellys) [inline .idea/.gitignore:1]: please. remove this file from the commit

**Review by ravellys — CHANGES_REQUESTED** [review]
remove the path .idea from the commit

File: Readme.md [hunk 1]
```diff
@@ -32,22 +32,27 @@ On Windows, run:
 pip install -r requirements.txt
 ```
 
-5. go to app path
+5. Environment variables
```
Comment (ravellys) [inline Readme.md:35]: 💯

File: .idea/.gitignore
Comment (evandrosystems) [reply .idea/.gitignore:1]: Ok!

Comment (evandrosystems) [reply .idea/.gitignore:1]: Done!

Comment (ravellys) [reply .idea/.gitignore:1]: awesome

**Review by ravellys — APPROVED** [review]
LGTM

**Pull request merged**